src/clj/rems/workflow/dynamic.clj

```diff
@@ -681,7 +754,8 @@
 (defn possible-commands
   "Returns the commands which the user is authorized to execute."
   [actor application-state]
-  (permissions/user-permissions application-state actor))
+  (-> (permissions/user-permissions application-state actor)
+      (conj ::accept-invitation)))
```

Comment:
hmm why cant user-permissions include this?

Reply:
It doesn't really support a set of permissions that are always available. It can technically be moved there of course but involves rewriting what that is in the first place. Perhaps @luontola can explain how it should be. I think we agreed on this place earlier.

Reply:
ok